src/main/resources/data/categories/Shipments.json

```diff
@@ -0,0 +1,4 @@
+ {
```

Comment:
Name of the file to be consistent with other filenames should start with lowercase letter`shipments.json`

Reply:
Done